spec/schema.json

```diff
@@ -0,0 +1,20 @@
+{
+  "openapi": "3.0.9",
+  "info": {
+    "title": "Companies House - Auth Code Notification",
+    "description": "",
```

Comment:
I would have added a simple description, something like `An API used to send email notifications when new auth code requested. `